GettingStarted.md

```diff
@@ -128,7 +147,7 @@ Alternatively, you can setup your scene manually as follows:
 > **For Hololens**
 > Optionally, if you wish to enable spatial mapping in your scene for HoloLens, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
 
-# 5 Building your project 
+# 7. Building your project 
 The MRTK provides you a quick and easy way to generate your Unity project from a custom window, which can be found under:
```

Comment:
Let's replace MRTK with HTK (throughout) in this branch. Thanks!

Reply:
I fixed it for this file. Let me know if I should fix it globally for this branch